docker/ubuntu-2404-builder.Dockerfile

```diff
@@ -90,8 +90,10 @@ RUN cd /root \
     mdbook-tabs             \
     sccache                 \
     wasm-pack               \
+    cargo-generate@0.22.0 \
```

Comment:
Is there a reason for using this version over the latest (0.23.1)?

Reply:
@James-Mart 's spec. Looks mostly like lib version bumping all the way up to 0.23.1